menu.json

```diff
@@ -529,93 +529,54 @@
-                            "navigateUrl": "https://www.ssw.com.au/ssw/Company/Offices/Sydney/Directions/",
-                            "text": "Map to SSW Sydney",
-                            "cssClass": "map"
-                        }
-                    ]
-                },
-                {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/Canberra.aspx",
-                    "navigateUrlOnMobileOnly": true,
-                    "text": "Canberra .NET UG",
-                    "children": [
                         {
                             "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/Canberra.aspx",
-                            "text": "Upcoming Sessions"
+                            "text": "Canberra .NET UG"
                         },
-                        {
-                            "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/UserGroupMapDirectionsCanberra.aspx",
-                            "text": "Map to Microsoft Canberra",
-                            "cssClass": "map"
-                        }
-                    ]
-                },
-                {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/Melbourne.aspx",
-                    "navigateUrlOnMobileOnly": true,
-                    "text": "Melbourne .NET UG",
-                    "breakListBefore": true,
-                    "children": [
                         {
                             "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/Melbourne.aspx",
-                            "text": "Upcoming Sessions"
+                            "text": "Melbourne .NET UG"
                         },
-                        {
-                            "navigateUrl": "https://www.ssw.com.au/ssw/Company/Offices/Melbourne/Directions/",
-                            "text": "Map to SSW Melbourne",
-                            "cssClass": "map"
-                        }
-                    ]
-                },
-                {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/Brisbane.aspx",
-                    "navigateUrlOnMobileOnly": true,
-                    "text": "Brisbane Full Stack UG",
-                    "children": [
                         {
                             "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/Brisbane.aspx",
-                            "text": "Upcoming Sessions"
-                        },
-                        {
-                            "navigateUrl": "https://www.ssw.com.au/ssw/Company/Offices/Brisbane/Directions/",
-                            "text": "Map to SSW Brisbane",
-                            "cssClass": "map"
+                            "text": "Brisbane Full Stack UG"
                         }
                     ]
                 },
+
                 {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/Live/",
+                    "navigateUrl": "ttps://www.ssw.com.au/ssw/Live/",
```

Comment:
Typo?

Reply:
Fixed in 38da1626153015489fdea06d2b281c52a06136e4